Data/ByteString/Short/Internal.hs

```diff
@@ -153,7 +158,9 @@ instance Ord ShortByteString where
     compare = compareBytes
 
 instance Semigroup ShortByteString where
-    (<>)    = append
+    (<>) = append
+    sconcat (x :| xs) = concat (x : xs)
+    stimes = stimesMonoid
```

Comment:
Why are the usual pragmas missing here?

Reply:
Because I don't understand the when and why for such `INLINE` incantations on instance method definitions clearly enough to want to guess whether or not they ought to be here.

Reply:
Please add `INLINABLE` pragmas then. This should ensure these methods can be properly specialized.